AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/OrganizationController.cs

```diff
@@ -69,8 +73,21 @@ public IActionResult Edit(OrganizationEditModel organization)
             
             if (ModelState.IsValid)
             {
-                int id = _mediator.Send(new OrganizationEditCommand { Organization = organization });
-                return RedirectToAction("Details", new { id = id, area = "Admin" });
+                var originalOrganization = _dataAccess.GetOrganization(organization.Id);
+                if (originalOrganization != null)
```

Comment:
Rather than re-introducing a dependency on `IAllReadyDataAccess` here, let's instead run with the Query/Handler pattern to validate.